pinot-plugins/pinot-minion-tasks/pinot-minion-builtin-tasks/src/main/java/org/apache/pinot/plugin/minion/tasks/BaseMultipleSegmentsConversionExecutor.java

```diff
@@ -152,6 +145,15 @@ protected void postProcess(PinotTaskConfig pinotTaskConfig) {
         }
       }
 
+      // Delete the input segments
+      // NOTE: Delete the input segments after tarring the converted segments to avoid deleting the converted segment
+      //       when the conversion happens in-place
```

Comment:
when does the conversion happen in-place? is that valid case? I wonder if some sanity check is needed instead of adjust the order here.

Reply:
+1 to better describe what "in-place" means: conversion task have the same original_segments/converted_segment  dir, thus delete the  original_segments dir will also delete the converted_segment dir. LGTM otherwise.

Reply:
We don't enforce the output to be different from the input, and for certain cases if no conversion needed, the executor can directly return the input dir. Will update the comment to reflect that